client/src/components/pages/database/database-view.tsx

```diff
@@ -1,11 +1,101 @@
 import "./database-view.module.css";
 import BreakdownChart from "./breakdown-chart";
+import CustomTable from "@/components/ui/table";
+
+
+interface RowWithSubRows {
+  rowData: [string, ...string[]]
+  subRowData?: [string, ...string[]][]
+}
```

Comment:
Changes requested: Define interface in one place. When it is defined in two places, it opens a risk for one changing, but not the other.

```suggestion
import RowWithSubRows from "@components/ui/table";
```